simca_osx.py

```diff
@@ -0,0 +1,120 @@
+import sys
+from optparse import OptionParser
+from subprocess import call
+from os import path
+
+resumeMode = False
+disableBloodlust = False
+optimalRaid = False
+scaleFactors = 0
+json2 = False
+scaleOnly = None
+
+def sim(profile, fight, iterations=10000, time=300, variance=0.1, bossCount=1, cores=4, outputFile=None, logFile=None, plotStats=None, plotPoints=20, plotStep=160.0, plotTargetError=0.1, xmlFile = None, jsonFile = None):
+    print("Revision 51 2017-05-11")
+
+    arguments='iterations={0} calculate_scale_factors={1} max_time={2} vary_combat_length={3} fight_style={4} desired_targets={5} threads={6}'.format(iterations, scaleFactors, time, variance, fight, bossCount, cores)
+    if disableBloodlust:
+        arguments+=' override.bloodlust=0'
+    if plotStats:
+        arguments+=' dps_plot_stat={0} dps_plot_points={1} dps_plot_step={2}'.format(plotStats, plotPoints, plotStep)
+        if plotTargetError != 0.1:
+            arguments+=' dps_plot_target_error={0}'.format(plotTargetError)
+    if scaleOnly:
+        arguments += ' scale_only={0}'.format(scaleOnly)
+    if outputFile:
+        arguments+=' html={0}.html'.format(outputFile+'_'+profile+'_'+fight+'_'+bossCount+'_'+time)
+    if xmlFile:
+        arguments+=' xml={0}.xml'.format(xmlFile+'_'+profile+'_'+fight+'_'+bossCount+'_'+time)
+    if jsonFile:
+        arguments+=' json2={0}.json'.format(jsonFile+'_'+profile+'_'+fight+'_'+bossCount+'_'+time)
+    arguments += ' output=nul '
+    arguments+=' {0}'.format(profile)
+    if logFile:
+       arguments+=' > {0}.log'.format(logFile)
+    hasJsonFile = resumeMode and jsonFile and path.isfile('{0}.json'.format(jsonFile+'_'+profile+'_'+fight+'_'+bossCount+'_'+time))
+    hasHtmlFile = resumeMode and outputFile and path.isfile('{0}.html'.format(outputFile+'_'+profile+'_'+fight+'_'+bossCount+'_'+time))
+
+    if hasHtmlFile:
+       print('Resume Mode! Skipping {0}\n'.format(outputFile+'_'+profile+'_'+fight+'_'+bossCount+'_'+time))
+    elif hasJsonFile:
+       print('Resume Mode! Skipping {0}\n'.format(jsonFile+'_'+profile+'_'+fight+'_'+bossCount+'_'+time))
+    else:
+       call("./simc "+arguments, shell=True)
```

Comment:
This is the only line that is changed from the original, not sure if this change works in windows